### PR TITLE
API-1637: Add the creation and update date of a Ref. entity Record

### PR DESCRIPTION
**Pull request by dFayet**
related to : https://github.com/akeneo/pim-enterprise-dev/pull/13211

### CODE REVIEW

**Review by LevFlavien — CHANGES_REQUESTED** [review]
Add readonly properties on PATCH routes

File: content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml [hunk 1]
```diff
@@ -24,6 +24,16 @@ properties:
             data:
               type: object
               description: Reference entity record value
+  created:
+      type: string
+      description: Date of creation
+      format: dateTime
+      default: null
```
Comment (LevFlavien) [inline content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml:31]: immutable property (see Product for example)

File: content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml [hunk 2]
```diff
@@ -24,6 +24,16 @@ properties:
...
+  update:
```
Comment (benjaminleneve) [inline content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml:32]: Replace update with updated

File: content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml (hunk 1 above)
Comment (benjaminleneve) [inline content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml:31]: Add read-only attribute for created and updated

**Review by benjaminleneve — APPROVED** [review]

File: content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml [hunk 3]
```diff
@@ -24,6 +24,19 @@ properties:
...
+  created:
+      type: string
+      description: Date of creation
+      format: dateTime
+      default: null
+      x-immutable: true
+      x-read-only: true
+  updated:
+      type: string
+      description: Date of the last update
```
Comment (LevFlavien) [inline content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml:36]: Add "(Only available on SaaS platforms)" to both descriptions

Comment (LevFlavien) [reply content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml:36]: FYI When v7 is released, all this kind of message will be cleaned up

**Review by SamirBoulil — APPROVED** [review]

**Pull request merged**